src/runtime_src/core/emulation/hw_em/generic_pcie_hal2/shim.cxx

```diff
@@ -571,9 +600,10 @@ namespace xclhwemhal2 {
           mLogStream << __func__ << " xocc command line: " << launcherArgs << std::endl;
 
         const char* simMode = NULL;
-        if ( pdi ) {
-          launcherArgs += " -pdi " + pdiFileName + " ";
+        if ( emuData ) {
```

Comment:
No spaces around if condition.

Reply:
updated as the per the review comments. Can you please push the changes. We are working on AIE design convergence. By Monday, we want all the changes to be in HEAD and need to work with bootgen team to take it further